node/actors/bft/src/leader/mod.rs

```diff
@@ -8,4 +8,8 @@ mod state_machine;
 #[cfg(test)]
 mod tests;
 
+#[allow(unused_imports)]
```

Comment:
Bikeshedding: IMO, using `#[cfg(test)]` on test-specific re-exports would make the intention clearer.

Reply:
Yes, I didn't understand why it doesn't detect the import 🤦